tests/unit_tests/datasets/segmentation/test_coco_segmentation_dataset.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+from torch.utils.data import DataLoader
+from data_gradients.managers.segmentation_manager import SegmentationAnalysisManager
+from data_gradients.datasets.segmentation.coco_segmentation_dataset import CocoSegmentationDataset
+
+
+class CocoSegmentationDatasetTest(unittest.TestCase):
+    def setUp(self):
+        self.train_set = CocoSegmentationDataset(root_dir="../../../../example_dataset/tinycoco", split="train", year="2017")
```

Comment:
Let's figure out path using `os.path.dirname(__file__)` and not assume that tests will be run from the same dir. 
If I run `python -m unit_test tests.unit_tests.datasets....` that would crash since cwd would be different.

Reply:
Changed in 92519f5f088612d83137a58c7b98e607c1075d9f